components/atom/statusSpot/src/index.js

```diff
@@ -0,0 +1,26 @@
+import React, {Component, PropTypes} from 'react'
+
+class AtomStatusSpot extends Component {
+  render () {
+    return (
+      <div className='sui-AtomStatusSpot'>
+        <span className={'sui-AtomStatusSpot-status sui-AtomStatusSpot-status-' + this.props.status} />
+      </div>
+    )
+  }
+}
+
+AtomStatusSpot.displayName = 'AtomStatusSpot'
+
+AtomStatusSpot.propTypes = {
+  /**
+   * The string content is the status spot
+   */
+  status: PropTypes.string
```

Comment:
Shouldn't this be `oneOf` instead of free string?

Reply:
You're right!!!